frontend/http/parser.go

```diff
@@ -99,10 +99,10 @@ func ParseScrape(r *http.Request) (*bittorrent.ScrapeRequest, error) {
 		Params:     qp,
 	}
 
-	return request, nil
+	return rs.SanitizeScrape(request)
 }
 
-// requestedIP determines the IP address for a BitTorrent client request.
+// requestedIP determines the IP address for a bt client request.
```

Comment:
Let's stick with `BitTorrent`, although I've read `bt` before, it doesn't make this easier to understand.